modules/verifying-serverless-app-deployment.adoc

```diff
@@ -7,14 +7,22 @@
 
 To verify that your serverless application has been deployed successfully, you must get the application URL created by Knative, and then send a request to that URL and observe the output.
 
-NOTE: If you changed the application name from `helloworld-go` to a new name when creating the `service.yaml` file, replace `helloworld-go` in the commands with the new name you created.
+[NOTE]
+====
+{ServerlessProductName} supports the use of both HTTP and HTTPS URLs, however the output from `oc get ksvc <SERVICE-NAME>` will always print URLs using the `http://` format.
+====
 
 .Procedure
 
+[NOTE]
+====
+]If you changed the application name from `helloworld-go` to a new name when creating the `service.yaml` file, replace `helloworld-go` in the commands with the new name you created.
```

Comment:
Redundant bracket at the beginning.

And I have no idea what this sentence means :slightly_smiling_face:

Reply:
I'd probably suggest that after getting the URL they can use whatever the URL is with "https://" and give an example

Reply:
After taking a look at the rendered docs. I think this whole note is unnecessary. It must be clear to everyone that the name will be different if they used something different.

And I'd suggest adding later below (similar to what we have for http)
curl https://helloworld-go.default.example.com